photon-client/src/integTest/scala/com/linkedin/photon/ml/data/GameConvertersTest.scala

```diff
@@ -14,10 +14,10 @@
  */
 package com.linkedin.photon.ml.data
 
+import org.apache.spark.sql.types._
```

Comment:
I was assuming that Scala would keep them in front, since that's a shorthand for multiple imports from one package, and that package comes first, but I just double-checked on my end and it looks like it doesn't.

Disregard